env.example.json

```diff
@@ -24,6 +24,10 @@
     "affiliateId": "xxxx",
     "affiliateMargin": 0.5
   },
+  "x_COINZARK_INIT": {
+    "apiKey": "xxxx",
+    "affiliateMargin": 0.5
```

Comment:
The plugin itself seems to be using `affiliateId` and `affiliateFee` here. Is this right?

Reply:
Good catch. This was indeed incorrect in the sample. Changed that.